app/assets/javascripts/hosts.show.js

```diff
@@ -101,8 +101,10 @@ app.controller('HostShowController', ['$scope', '$http', function($scope, $http)
 	}
 
 	$scope.closeEvening = function() {
-		$scope.host.strangers = false;
-		$scope.save();
+		if (confirm('האם אתה בטוח?')) {
```

Comment:
can you get the value of the locale and if no he, ask in english?

Reply:
Done.